modules/core/include/opencv2/core/ocl.hpp

```diff
@@ -278,55 +278,52 @@ class CV_EXPORTS Platform
     Impl* p;
 };
 
-/*
-//! @brief Attaches OpenCL context to OpenCV
+/**
+// @brief Attaches OpenCL context to OpenCV
```

Comment:
Full list of doxygen supported comments is here (for reference): https://www.stack.nl/~dimitri/doxygen/manual/docblocks.html